cmd/status/root_test.go

```diff
@@ -87,3 +91,40 @@ func TestStatusCmd(t *testing.T) {
 		})
 	}
 }
+
+func authInterceptor(ctx context.Context, req any, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (resp any, err error) {
```

Comment:
As this is just one way to do auth, I bet we will get tons of requests to add more once we add this one :sweat_smile: 
Let's discuss this offline next week.

Reply:
Never mind, this is just test code :sweat_smile: